data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/export/ExportTaskManager.java

```diff
@@ -46,12 +50,17 @@ public String submitExportJob(String tableArn, String bucket, String prefix, Str
 
 
         try {
+            dynamoAggregateMetrics.getExportApiInvocations().increment();
             ExportTableToPointInTimeResponse response = dynamoDBClient.exportTableToPointInTime(req);
 
             String exportArn = response.exportDescription().exportArn();
             String status = response.exportDescription().exportStatusAsString();
             LOG.debug("Export Job submitted with ARN {} and status {}", exportArn, status);
             return exportArn;
+        } catch (final InternalServerErrorException e) {
+            dynamoAggregateMetrics.getExport5xxErrors().increment();
```

Comment:
should we also consider adding pipeline specific metric ?

Reply:
I don't think there's any value in that for these metrics at least. There may be other pipeline specific metrics we can add, but that can be a future PR